food_trucks/demo/pos_config.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="pos_config_2" model="pos.config">
+    <field name="name">Food Truck</field>
+    <field name="tip_product_id" eval="False"/>
+    <field name="self_ordering_pay_after">each</field>
+  </record>
```

Comment:
I only see one PoS config in the template. I think you only need to edit the name of the config created with the scenario below. 
```suggestion
```